src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -103,6 +104,14 @@ public class ExecutionPlanGraphPropertyBase
         /// </summary>
         /// <value></value>
         public string DisplayValue { get; set; }
+        /// <summary>
+        /// Indicates what kind of value is better amongst 2 values of the same property
+        /// </summary>
+        public BetterValue BetterValue { get; set; }
```

Comment:
What does "Better" mean in this context? Does it refer to something being greater/optimal, and is the value referring to the display value or a different value?

Reply:
I think better means more optimal value in this case. The value refers to the value of the property. Display value is just the value with some additional formatting.